### ISSUE TITLE
Bad bootstrap_config.sh

### ISSUE THREAD

**Issue by mjwall** [apache/accumulo#994]
In 1.9.2, I ran `./bin/bootstrap_config.sh ` and chose '512' and 'standalone'.  Tserver fails to start with 

```
2019-02-27 12:45:49,681 [tserver.TabletServer] ERROR: Uncaught exception in TabletServer.main, exiting
java.lang.IllegalArgumentException: Maximum tablet server map memory 83,886,080 block cache sizes 24,117,248 and mutation queue size 52,428,800 is too large for this JVM configuration 129,761,280
	at org.apache.accumulo.tserver.TabletServerResourceManager.<init>(TabletServerResourceManager.java:186)
	at org.apache.accumulo.tserver.TabletServer.<init>(TabletServer.java:408)
	at org.apache.accumulo.tserver.TabletServer.main(TabletServer.java:3206)
	at org.apache.accumulo.tserver.TServerExecutable.execute(TServerExecutable.java:33)
	at org.apache.accumulo.start.Main$1.run(Main.java:130)
	at java.lang.Thread.run(Thread.java:748)
```

Setting tserver.memory.maps.max to 30M in the accumulo-site.xml got past the issue

**Comment by ctubbsii** [comment]
This script was removed in 2.0 in favor of improved self-documented example configs. So, this only needs to be fixed in 1.9.3.

**Comment by ctubbsii** [comment]
PR in #1055
